doc/c64.sgml

```diff
@@ -216,6 +250,9 @@ configuration.
   palette of the 16 C64 colors).
 </descrip><p>
 
+Note that the graphics drivers are incompatible with the
+<tt/c64-ram.emd (c64_ram_emd)/ extended memory driver and the
+ <tt/c64-soft80.o/ software 80 columns conio driver.
```

Comment:
>> [...] use soft80 and TGI together?

> it shouldnt be terribly hard to make both work together... later, perhaps :=)

I see. I wasn't suggesting this. I just wanted to find out if I understood it right that from the VIC perspective both things are the same...